### ISSUE TITLE
Can't operate hamburger menu using keyboard

### ISSUE THREAD

**Issue by ssciolla** [tl-its-umich-edu/my-learning-analytics#804]
Thank you for contributing to this project!

- Make sure to search the [issues](https://github.com/tl-its-umich-edu/my-learning-analytics/issues) for duplicates first!

## Expected behavior (A clear and concise description of what you expected to happen) : 

Users should be able to navigate through and select the options in the hamburger menu using the keyboard (Tab and Enter).

## Describe the bug (Tell us what happens instead of the expected behavior) :

From 10/28/19 accessibility review:

"Hamburger menu items cannot be operated via the keyboard. This is a minor issue because the items can be accessed via keyboard via other means. But the user needs to discover those means. ... Some users will not be able to navigate using the hamburger menu"

## Additional context (Add any other context about the problem here) :

The review makes the following observation: "The [Material Design] component is at fault I imagine, as the demos are also problematic."

Solving this will likely require some research to see how other teams and projects have addressed this issue.

**Comment by jennlove-um** [comment]
Testing passes.  I am able to Tab to access the Hamburger menu, then enter to open it, then tab and enter to select an option.

**Comment by gsilver** [comment]
@jennlove-um - ship it!
